snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -16,6 +16,18 @@ init_config:
   #
   # ignore_nonincreasing_oid: false
 
+  ## @param oid_batch_size - integer - optional - default: 10
+  ## Number of OIDs handled by each batch. Performance can be improved by increasing this number, though
+  ## it will also use more resource.
+  #
+  # oid_batch_size: 10
+
+  ## @param shared_mib_builder - boolean - optional - default: false
+  ## Set to true to share MIB builders between instance. It will reduce memory
+  ## usage, but doesn't work in some edge cases.
```

Comment:
maybe add a note about how does the user know if it works or not (is there an error when it does not work?)